control/serializers.py

```diff
@@ -97,10 +97,11 @@ class Meta:
 
 class QuestionUpdateSerializer(serializers.ModelSerializer):
     id = serializers.IntegerField(required=False)
+    order = serializers.CharField(required=False)
```

Comment:
Just a detail: not sure we need to specify this here, since the field exists in the mode.

Reply:
Ha ok : `required=Fase`.